roles/install_ansible_rulebook/tasks/main.yml

```diff
@@ -1,3 +1,54 @@
 ---
-- name: "Install for {{ ansible_distribution }}"
-  ansible.builtin.include_tasks: "install_{{ ansible_distribution | lower }}.yml"
+- name: Include OS specific vars
+  ansible.builtin.include_vars:
+    file: "{{ ansible_os_family }}.yml"
+
+- name: Ensure dependencies are installed
+  become: "{{ 'false' if ansible_distribution == 'MacOSX' else 'true' }}"
+  ansible.builtin.package:
+    name: "{{ package_list }}"
+
+- name: Set required facts
+  ansible.builtin.set_fact:
+    _shell: "{{ ansible_user_shell | basename }}"
+
+- name: Include MacOSX specific tasks
+  ansible.builtin.include_tasks: tasks_macosx.yml
+  when: ansible_distribution == "MacOSX"
+
+- name: Set JAVA_HOME env var
+  when: _shell != ""
+  ansible.builtin.lineinfile:
+    path: "~/.{{ _shell }}rc"
+    regexp: "^(export )*JAVA_HOME"
+    line: "export JAVA_HOME={{ java_home_path }}"
+    mode: "0660"
+    create: true
+
+- name: Upgrade pip
+  become: true
+  ansible.builtin.pip:
+    name: pip
+    state: latest # noqa package-latest
+
+- name: Install pip dependencies
+  ansible.builtin.pip:
+    name:
+      - ansible
+      - ansible-runner
+      - wheel
+    state: latest # noqa package-latest
+
+- name: Install ansible-rulebook
+  environment:
+    PIP_NO_BINARY: jpy
+    JAVA_HOME: "{{ java_home_path }}"
+  ansible.builtin.pip:
+    name: ansible-rulebook
+    state: latest # noqa package-latest
+
+- name: Install community.general collection
+  ansible.builtin.command: ansible-galaxy collection install community.general
```

Comment:
I figured it was unnecessary since our docs state to install the collection and run the playbook via `ansible.eda.install_rulebook_cli`. But I guess it does not hurt to include it in the final step in case a user clones the repo and runs the role "directly". Ill add it back.

Reply:
The playbook can be used with hosts that do not have installed the collection. Otherwise the installation of ansible would be also unnecessary.